src/lib/Form/TrashLocationOptionProvider/CanHaveDrafts.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Form\TrashLocationOptionProvider;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Location;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class CanHaveDrafts implements TrashLocationOptionProvider
+{
+    private TranslatorInterface $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function supports(Location $location): bool
+    {
+        return true;
+    }
+
+    public function addOptions(FormInterface $form, Location $location): void
+    {
+        $form
+            ->add('can_have_drafts', ChoiceType::class, [
+                'label' =>
+                    /** @Desc("Drafts") */
+                    $this->translator->trans('drafts.list', [], 'ibexa_drafts'),
+                'help_multiline' => [
+                    /** @Desc("Sending this content item to Trash will also delete all drafts of content items that haven’t been published yet, and belong to the trashed subtree.") */
+                    $this->translator->trans('trash.modal.send_to_trash_draft_warning.message', [], 'messages'),
```

Comment:
is messages the proper domain?

Reply:
@dew326 It directly corresponds to the similar provider:
https://github.com/ibexa/admin-ui/blob/4.6/src/lib/Form/TrashLocationOptionProvider/HasChildren.php#L53

Reply:
In the new packages we tend to walk away from `messages`, true but not sure about `AdminUI` and its leftovers here and there. Maybe `ibexa_drafts` is more suitable domain here, don't you mind @barw4 regenerating translations accordingly?

Reply:
Resolved in https://github.com/ibexa/admin-ui/pull/1321/commits/f52dd78c63e49a16402f07e3f8b3a97c5420048e